common/metrics/defs.go

```diff
@@ -201,8 +201,8 @@ const (
 	PersistenceCompleteTimerTaskScope
 	// PersistenceRangeCompleteTimerTaskScope tracks CompleteTimerTasks calls made by service to persistence layer
 	PersistenceRangeCompleteTimerTaskScope
-	// PersistenceCreateTaskScope tracks CreateTask calls made by service to persistence layer
-	PersistenceCreateTaskScope
+	// PersistenceCreateTasksScope tracks CreateTask calls made by service to persistence layer
```

Comment:
Not necessarily a problem, but what was the thinking behind this name change?

Reply:
Unifying the approach of variable naming/variable to method mappings. To be able to generate the code, there should be a simple rule to determine the scope. I've changed some names and added a couple of scopes, keeping them mapped to the same operation in the metric results. But still ended up with one ugly.
```
{{ if and (eq $interfaceName "VisibilityManager") (eq $methodName "DeleteWorkflowExecution") -}}
                {{ $scopeName = "metrics.PersistenceVisibilityDeleteWorkflowExecutionScope" }}
            {{ end -}}
```

Reply:
totally makes sense. It took me a little while to understand, but I can see how the concistency makes it better for templating